### PR TITLE
Add details to How to Join RDS

### PR DESCRIPTION
**Pull request by sahsisunny**
### Issue
#373 

### What is the change?

- Added "How to Join Real Dev Squad" details
- Three Files have been changed index.html, main.css and index.js.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
Before :
before

After:
https://user-images.githubusercontent.com/70854507/211705526-24700700-c526-40ec-81a8-f3f165210679.mp4

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -172,13 +172,9 @@ body {
   text-align: center;
 }
 
-.join-description {
-  padding: 0 150px 32px;
-  font-weight: 400;
-  line-height: 175%;
-  text-align: justify;
+.desc-join {
+  color: var(--color-text-secondary);
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply css/main.css:176]: No it is not in use now.

File: css/main.css [hunk 2]
```diff
@@ -172,11 +172,38 @@ body {
...
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 34px;
+  color: var(--color-text);
+}
+
+.btn-join:hover {
+  background: #6dc66d;
```
Comment (RitikJaiswal75) [inline css/main.css:205]: lets create a variable for this hex

**Review by RitikJaiswal75 — COMMENTED** [review]
let's keep this `how to join` section behind the feature flag as of now once the three-step signup flow is integrated we can remove the feature flag

File: css/main.css (hunk 2 above)
Comment (sahsisunny) [reply css/main.css]: Done

File: index.html [hunk 3]
```diff
@@ -246,19 +246,19 @@ <h1 class="description-title">What is Real Dev Squad?</h1>
         </div>
       </section>
 
-      <section class="section-features hide" id="section-join">
+      <section class="section-features" id="section-join">
         <h1 class="features-title">How to Join Real Dev Squad</h1>
```
Comment (rohan09-raj) [inline index.html:250]: According to the design, the text should be `How can you join?` here

File: js/index.js [hunk 4]
```diff
@@ -101,3 +101,8 @@ modalTriggers.forEach((trigger) => {
     });
   });
 });
+
+const joinBTN = document.getElementById('btn-join');
```
Comment (rohan09-raj) [inline js/index.js:105]: Can we name it - `joinBtn` ?

File: css/main.css [hunk 5]
```diff
@@ -172,11 +174,38 @@ body {
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 34px;
+  color: var(--color-text);
+}
+
+.btn-join:hover {
+  background: var(--color-button-hover);
+  cursor: pointer;
```
Comment (rohan09-raj) [inline css/main.css:208]: Is the content responsive? Can we please have a screenshot of mobile and tablet views as well?

File: css/main.css [hunk 6]
```diff
@@ -172,11 +174,38 @@ body {
...
+.btn-join {
+  width: 22.5rem;
+  height: 5.25rem;
+  background: var(--color-button);
+  border-radius: 10px;
+  border: none;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 36px;
```
Comment (rohan09-raj) [inline css/main.css:201]: We can use shorthand of `font`

File: css/main.css [hunk 7]
```diff
@@ -172,11 +174,38 @@ body {
...
+  align-items: center;
+}
+
+.desc-join p {
+  font-family: 'Roboto';
+  font-style: normal;
   font-weight: 400;
-  line-height: 175%;
-  text-align: justify;
+  font-size: 20px;
```
Comment (rohan09-raj) [inline css/main.css:188]: Can we shorthand of `font` here?

File: js/index.js (hunk 4 above)
Comment (sahsisunny) [reply js/index.js:105]: Yes, Done

File: css/main.css (hunk 5 above)
Comment (sahsisunny) [reply css/main.css]: Please watch the video and comment, if media queries are required or not
https://user-images.githubusercontent.com/70854507/211486574-d8c40972-9e37-4df6-a0a7-f16de6cc5b53.mp4

**Comment by sahsisunny** [comment]
> 

Feature flag is already written i am add only hide class to this section

File: css/main.css (hunk 5 above)
Comment (sahsisunny) [reply css/main.css]: added a some media queries for button please check

Comment (rohan09-raj) [reply css/main.css]: Thanks

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**